src/Npgsql/NpgsqlConnection.cs

```diff
@@ -993,7 +993,25 @@ public int ProcessID
         /// <remarks>
         /// See https://www.postgresql.org/docs/current/static/sql-copy.html.
         /// </remarks>
-        public NpgsqlBinaryImporter BeginBinaryImport(string copyFromCommand)
+        public NpgsqlBinaryImporter BeginBinaryImport(string copyFromCommand) =>
+            BeginBinaryImport(copyFromCommand, async: false, CancellationToken.None).GetAwaiter().GetResult();
+
+        /// <summary>
+        /// Begins a binary COPY FROM STDIN operation, a high-performance data import mechanism to a PostgreSQL table.
+        /// </summary>
+        /// <param name="copyFromCommand">A COPY FROM STDIN SQL command</param>
+        /// <param name="cancellationToken">An optional token to cancel the asynchronous operation. The default value is None.</param>
+        /// <returns>A <see cref="NpgsqlBinaryImporter"/> which can be used to write rows and columns</returns>
+        /// <remarks>
+        /// See https://www.postgresql.org/docs/current/static/sql-copy.html.
+        /// </remarks>
+        public Task<NpgsqlBinaryImporter> BeginBinaryImportAsync(string copyFromCommand, CancellationToken cancellationToken = default)
```

Comment:
`ValueTask` for public API is more preferrable because it allows to change internals without breaking the public surface.

Reply:
Oh, okay, I'll keep that in mind.